### ISSUE TITLE
css-logical-props

### ISSUE THREAD

**Issue by Nemo64** [anandthakker/doiuse#102]
It seems that it logical properties are currently not detected.

I expect this to warn me about missing support:
`echo "body {padding-inline-start: 5px}"|doiuse --browsers "ie >= 9"`
https://caniuse.com/#feat=css-logical-props

**Comment by pkuczynski** [comment]
I would happily accept a PR if you have some time to work on it?

**Comment by clshortfuse** [comment]
Fixed in #159 

````
 echo "body {padding-inline-start: 5px}"| npx doiuse --browsers "ie >= 9"
<streaming css input>:1:1: CSS Logical Properties not supported by: IE (9,10,11) (css-logical-props)
````
